### ISSUE TITLE
No examples will work

### ISSUE THREAD

**Issue by jstansbe** [react-native-community/react-native-tab-view#507]
### Current behaviour

Using the code template from TopBarTextExample doesnt show any views. 

### Expected behaviour

Upon clicking a tab header, show the respective view.

### Code sample

```
import * as React from 'react';
import { StyleSheet, Dimensions, View } from 'react-native';
import { TabViewAnimated, TabBar, SceneMap } from 'react-native-tab-view';
import SimplePage from './SimplePage';




const Albums = () => <View style={[ styles.container, { backgroundColor: '#ff4081' } ]}></View>;
const Contacts = () => <View style={[ styles.container, { backgroundColor: '#ff4081' } ]}></View>;
const Article = () => <View style={[ styles.container, { backgroundColor: '#ff4081' } ]}></View>;
const Chat = () => <View style={[ styles.container, { backgroundColor: '#ff4081' } ]}></View>;


// type State = NavigationState<
//   Route<{
//     key: string,
//     title: string,
//   }>
// >;

const initialLayout = {
  height: 0,
  width: Dimensions.get('window').width,
};

export default class TopBarTextExample extends React.Component {
    static title = 'Scrollable top bar';
    static backgroundColor = '#3f51b5';
    static appbarElevation = 0;
  
    state = {
      index: 1,
      routes: [
        { key: 'article', title: 'Article' },
        { key: 'contacts', title: 'Contacts' },
        { key: 'albums', title: 'Albums' },
        { key: 'chat', title: 'Chat' },
      ],
    };
  
    _handleIndexChange = index =>
      this.setState({
        index,
      });
  
    _renderHeader = props => (
      <TabBar
        {...props}
        scrollEnabled
        indicatorStyle={styles.indicator}
        style={styles.tabbar}
        tabStyle={styles.tab}
        labelStyle={styles.label}
      />
    );
  
    _renderScene = SceneMap({
      albums: Albums,
      contacts: Contacts,
      article: Article,
      chat: Chat,
    });
  
    render() {
      return (
        <TabViewAnimated
          style={[styles.container, this.props.style]}
          navigationState={this.state}
          renderScene={this._renderScene}
          renderHeader={this._renderHeader}
          onIndexChange={this._handleIndexChange}
          initialLayout={initialLayout}
        />
      );
    }
  }
  
  const styles = StyleSheet.create({
    container: {
      flex: 1,
    },
    tabbar: {
      backgroundColor: '#3f51b5',
    },
    tab: {
      width: 120,
    },
    indicator: {
      backgroundColor: '#ffeb3b',
    },
    label: {
      color: '#fff',
      fontWeight: '400',
    },
  });
```

### Screenshots (if applicable)

![screenshot_1524164344](https://user-images.githubusercontent.com/1250522/39012362-3ae5fb06-43e2-11e8-9f70-b2eb163e5653.png)

### What have you tried

I've tried practically ALL of the examples and all have the same issue i've described. 
https://github.com/react-native-community/react-native-tab-view/tree/master/example

### Your Environment

| software              | version
| --------------------- | -------
| android        | 8.0.0
| react-native          | 0.55.3
| react-native-tab-view | 0.0.77
| node                  | 8.9.4
| yarn           |  1.3.2

**Comment by jstansbe** [comment]
Downgrading version to 0.074 solved my issues!
Closing this!

**Comment by donnes** [comment]
I think is needed to reopen this issue because the problem was not solved in v1.0.0

**Comment by satya164** [comment]
Can you provide an example on snack.expo.io?

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.

**Comment by quicksilverr** [comment]
Please reopen this issue, as it doesn't address the solution for it
